setup.py

```diff
@@ -61,6 +69,8 @@ def load_requirements(*requirements_paths):
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
No 3.5 and 3.7?

Reply:
We're currently not anticipating making an Open edX release that uses 3.5, and Python 3.7 support will probably come a little later.  I'm fine with adding tests and support for 3.7, but that's not on the critical path and it should probably be its own PR (especially if any code changes are needed).

Reply:
Thanks ... Very little incompatibility between 3.6 and 3.7 but the largest difference is that __async__ and __await__ are reserved words.

Reply:
For edx the biggest Py37 issue seems to be the tightening of the screws on __StopIteration__ which requires current versions of pylint.